StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -166,4 +166,10 @@ import Foundation
 
     // MARK: - PaymentSheet Force Success
     case paymentSheetForceSuccess = "mc_force_success"
+
+    // MARK: - PaymentSheet initialization
+    case paymentSheetStartedLoading = "mc_started_loading"
+    case paymentSheetDismissed = "mc_dismiss"
```

Comment:
Why is dismiss under `// MARK: - PaymentSheet initialization`?  It's not related to initialization, is it?

Reply:
A dismiss event can happen during initialization, but can obviously happen after init. I moved it to under it's own `MARK:`.